src/psyclone/dynamo0p3.py

```diff
@@ -1465,6 +1465,17 @@ def initialise(self, parent):
                                 "STENCIL_1D" + direction.upper() +
                                 ","+self.extent_value(arg)+")"))
                         parent.add(if_then)
+                elif stencil_type == "cross2d":
+                    # raise GenerationError(
```

Comment:
(Oh, and in case you didn't know, adding the line `import pdb; pdb.set_trace()` will stop execution at the current line and open the Python debugger for you.)

Reply:
Thanks for the info, and done.